content/documentation/pages/2-concepts/3-streams.md

```diff
@@ -6,11 +6,14 @@ description: 'Stream Processing Framework and Concepts'
 
 # Stream Processing
 
-Stream processing, is defined as the processing of infinite amount of data without interaction or interruption.
-Use cases for stream processing are real-time credit card fraud detection or predictive analytics.
-
-Stream processing in Data Flow is implemented architecturally as a collection of independent event-driven streaming applications that connect using messaging middleware, for example RabbitMQ or Apache Kafka.
-The collection of application is referred to as a streaming data pipelines.
+Stream processing is defined as the processing of an unbounded amount of data without interaction or interruption.
+Business cases for stream processing include:
+- Real-time credit card fraud detection or predictive analytics.
```

Comment:
It'd be nice to remove `.` from the statement

Reply:
I personally believe that no bullets should ever have periods.  However the pattern in the documentation as it stands does utilize periods on bullet statements.  Should I remove all of them?